src/store/getters.ts

```diff
@@ -1,10 +1,12 @@
 import { CalChartState } from '.';
 import StuntSheet from '@/models/StuntSheet';
 import { GetterTree } from 'vuex';
+import Show from '@/models/Show';
 
 const getters: GetterTree<CalChartState, CalChartState> = {
   // Show
   getShowTitle: (state): string => state.show.title,
+  getShow: (state): Show => state.show,
```

Comment:
Since `show` is already a property of `state`, it's simpler to not define a getter here. Instead, use something like `this.$store.state.show` in the components.

Reply:
Ping on this comment ^

Reply:
oops thanks